engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -113,7 +113,8 @@
   public static final String KEY_TAB_FILE = StramUserLogin.DT_AUTH_PREFIX + "store.keytab";
   public static final String TOKEN_ANTICIPATORY_REFRESH_FACTOR = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.factor";
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
-
+  public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
+  public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
```

Comment:
@PramodSSImmaneni IMO, those constants belong to org.apache.apex.api package. It should be possible to create Configuration programmatically and add these properties to the Configuration and launch an application using Launcher. If you agree, can you file JIRA?